content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -281,7 +281,7 @@ curl -X DELETE  'localhost:8080/apis/extensions/v1beta1/namespaces/default/repli
 Once the original is deleted, you can create a new ReplicaSet to replace it.  As long
 as the old and new `.spec.selector` are the same, then the new one will adopt the old Pods.
 However, it will not make any effort to make existing Pods match a new, different pod template.
-To update Pods to a new spec in a controlled way, use a [rolling update](#rolling-updates).
+To update Pods to a new spec in a controlled way, use a [Deplyoment](/docs/concepts/workloads/controllers/deployment/#creating-a-deployment), as ReplicaSets do not support a rolling update directly.
```

Comment:
```suggestion
To update Pods to a new spec in a controlled way, use a [Deployment](/docs/concepts/workloads/controllers/deployment/#creating-a-deployment), as ReplicaSets do not support a rolling update directly.
```

(or maybe wrap it over 2 lines)

Reply:
How do you suggest a wrap? Just a line break before "Deployment?"

Reply:
Sure, that'd work.

Reply:
Donezo. Thanks for looking. Sorry for the trouble